src/kili/entrypoints/mutations/data_connection/__init__.py

```diff
@@ -99,6 +100,11 @@ def synchronize_cloud_storage_connection(
 
         Returns:
             A dict with the DataConnection ID.
+
+        !!! warning "Azure Blob Storage"
+            This method currently does not work for Azure cloud storage connection using credentials mode.
+            Use service account mode instead or use the Kili app to synchronize the data connection.
```

Comment:
what is the service account mode ?

Reply:
https://docs.kili-technology.com/docs/creating-an-azure-blob-storage-integration#add-service-account

Reply:
Maybe we can add this link in the docstring?

Reply:
ah ah missed that